pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -200,7 +220,20 @@ func instrumentScanDoneEvent(invocationCtx workflow.InvocationContext, input wor
 	ic.SetTestSummary(toTestSummary(scanDoneEvent.Data.Attributes.UniqueIssueCount, scanDoneEvent.Data.Type))
 	ic.AddExtension("device_id", scanDoneEvent.Data.Attributes.DeviceId)
 
-	return nil
+	data, err := analytics.GetV2InstrumentationObject(ic)
+	if err != nil {
+		return nil, err
+	}
+
+	v2InstrumentationData := utils.ValueOf(json.Marshal(data))
```

Comment:
Please do not discard errors. If propagating this error isn't the right thing, let's debug-log it.

```suggestion
	v2InstrumentationData, err := json.Marshal(data)
	if err != nil {
		return nil, fmt.Errorf("failed to marshal instrumentation: %w", err)
	}
```

Reply:
copy pasted, the error is now handled.